### PR TITLE
Fix usability of isPayerActive method

### PR DESCRIPTION
**Pull request by iyiolapeter**

### CODE REVIEW

File: src/disbursements.ts [hunk 1]
```diff
@@ -175,9 +175,9 @@ export default class Disbursements {
   public isPayerActive(
     id: string,
     type: PartyIdType = PartyIdType.MSISDN
-  ): Promise<string> {
+  ): Promise<{ result: boolean }> {
```
Comment (ernest-okot) [inline src/disbursements.ts:178]: Why not change the return type this simply boolean instead of the object with the boolean?

Comment (iyiolapeter) [reply src/disbursements.ts:178]: Oh okay, I completely agree, will make the changes

File: src/disbursements.ts [hunk 2]
```diff
@@ -175,9 +175,10 @@ export default class Disbursements {
...
     id: string,
     type: PartyIdType = PartyIdType.MSISDN
-  ): Promise<string> {
+  ): Promise<boolean> {
     return this.client
-      .get<string>(`/disbursement/v1_0/accountholder/${type}/${id}/active`)
-      .then(response => response.data);
+      .get<{ result: boolean }>(`/disbursement/v1_0/accountholder/${String(type).toLowerCase()}/${id}/active`)
+      .then(response => response.data)
+      .then(data => data.result ? data.result : false)
```
Comment (ernest-okot) [inline src/disbursements.ts:182]: Is the result actually a Boolean, do we need the cast it?

**Review by ernest-okot — APPROVED** [review]
Looks good to me, thank you @iyiolapeter

**Comment by iyiolapeter** [comment]
Hi @ernest-okot, is it okay to exclude the examples folder from NYC coverage?

**Comment by ernest-okot** [comment]
> Hi @ernest-okot, is it okay to exclude the examples folder from NYC coverage?

Good idea. Look at this https://stackoverflow.com/a/42115872/4054434

**Comment by ernest-okot** [comment]
@iyiolapeter feel free to merge this if it's all done

**Review by ernest-okot — APPROVED** [review]

**Comment by iyiolapeter** [comment]
@ernest-okot it's all done but I can't complete the merge without write access. Can you help with the merge instead?

**Pull request merged**